src/languages/en.ts

```diff
@@ -579,6 +579,8 @@ const translations = {
         network: 'Network',
         reportID: 'Report ID',
         longID: 'Long ID',
+        entryID: 'Entry ID',
```

Comment:
I think we actually want to just use `Withdrawal ID`. We changed that but I don't think the mocks were updated in the doc